sos/report/plugins/system.py

```diff
@@ -33,5 +33,11 @@ def setup(self):
             "/proc/sys/net/ipv6/neigh/*/base_reachable_time"
         ])
 
+        # collect glibc tuning decisions
+        self.add_cmd_output([
+            "/usr/bin/ld.so --help",
```

Comment:
Collecting `--help` output seems strange. Is there a particular reason?

Reply:
I had the same question to the original reporter, quoting his answer:

> --help output contains some diagnostic output, too, perhaps in a more digestable form. It shows the search paths being used.